py/selenium/webdriver/remote/webdriver.py

```diff
@@ -82,11 +87,11 @@ def __init__(self, command_executor='http://127.0.0.1:4444/wd/hub',
         if type(self.command_executor) is bytes or isinstance(self.command_executor, str):
             self.command_executor = RemoteConnection(command_executor, keep_alive=keep_alive)
         self._is_remote = True
-        self.session_id = None
-        self.capabilities = {}
+        self.session_id = session_id
+        self.capabilities = capabilities
         self.error_handler = ErrorHandler()
         self.start_client()
-        self.start_session(desired_capabilities, browser_profile)
+        self.start_session(desired_capabilities, browser_profile, force_new=False)
```

Comment:
Hi, thanks for the comment by the way, this was my logic:

The idea was to maintain backwards compatibility, this the scenario i was thinking.

with the current implementation, if someone calls remote.start_session (and you are right this is a mostly private method, but you never know how someone could be using it) they will generate a new session_id, if i don't include this force_new=True  in this implementation, when that same user calls that method it will not get a new session_id, and it will break there code.

This is un unlikely scenario, and probably a little convoluted, in my opinion this makes sense, but i'm not attach to that particular code so if this needs to go in order for this code to be accepted, i will change it in a heart beat, no problem at all (i more interested on the chance to reuse the session_id).

let me know if it makes sense, or you'll like it to be changed